nncf/torch/checkpoint_loading.py

```diff
@@ -268,8 +268,26 @@ def run(self) -> Dict[str, torch.Tensor]:
         normalized_keys_to_load = NormalizedKeys(list(self.state_dict_to_load.keys()))
 
         has_legacy_storage_keys = False
+
+        has_version_agnostic_names = False
```

Comment:
please move this functionality to `NormalizedKeys` class as it was done in my PR for legacy_storage_keys: 
https://github.com/openvinotoolkit/nncf/pull/682/files#diff-40c3a1a297ae11ac63166f5c9a2117bb25bd2e0c548e6d36e07edf64bd9848dfR241

Reply:
The matching done in this PR cannot be done in `NormalizedKeys` because it requires both the model and the checkpoint key sets - we don't know whether `RELU` in a given model corresponds to `relu` or `relu_`, so we have to iterate over alternatives.